aws/resource_aws_backup_selection.go

```diff
@@ -162,6 +169,21 @@ func resourceAwsBackupSelectionDelete(d *schema.ResourceData, meta interface{})
 	return nil
 }
 
+func resourceAwsBackupSelectionImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	idParts := strings.Split(d.Id(), "|")
+	if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+		return nil, fmt.Errorf("unexpected format of ID (%q), expected <plan-id>:<selection-id>", d.Id())
```

Comment:
The delimiter above is `|` not `:`

```suggestion
		return nil, fmt.Errorf("unexpected format of ID (%q), expected <plan-id>|<selection-id>", d.Id())
```